mlflow/server/js/src/i18n/default/en.json

```diff
@@ -207,6 +207,10 @@
     "defaultMessage": "{timeSince, plural, =1 {1 minute} other {# minutes}} ago",
     "description": "Text for time in minutes since given date for MLflow views"
   },
+  "7kUi8J": {
+    "defaultMessage": "Duration:",
+    "description": "Row title for the duration of runs on the experiment compare runs page"
+  },
```

Comment:
Might have to run the i18n extract script again to add `unknown` to the default strings here